src/main/java/org/folio/inventory/validation/ItemMarkAsMissingValidators.java

```diff
@@ -0,0 +1,35 @@
+package org.folio.inventory.validation;
+
+import static java.util.EnumSet.of;
+
+import java.util.EnumSet;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.inventory.domain.items.Item;
+import org.folio.inventory.domain.items.ItemStatusName;
+import org.folio.inventory.exceptions.UnprocessableEntityException;
+import org.folio.inventory.support.CompletableFutures;
+import org.folio.inventory.support.http.server.ValidationError;
+
+public final class ItemMarkAsMissingValidators {
+  private static final EnumSet<ItemStatusName> ALLOWED_STATUS_TO_MARK_MISSING = of(
+    ItemStatusName.AVAILABLE,
+    ItemStatusName.IN_TRANSIT,
+    ItemStatusName.IN_PROCESS,
+    ItemStatusName.AWAITING_PICKUP,
+    ItemStatusName.AWAITING_DELIVERY,
+    ItemStatusName.WITHDRAWN,
+    ItemStatusName.PAGED);
+
+  private ItemMarkAsMissingValidators() {}
+
+  public static CompletableFuture<Item> itemHasAllowedStatusToMarkAsMissing(Item item) {
+    if (ALLOWED_STATUS_TO_MARK_MISSING.contains(item.getStatus().getName())) {
+      return CompletableFuture.completedFuture(item);
+    }
+
+    return CompletableFutures.failedFuture(new UnprocessableEntityException(
```

Comment:
Can this can be statically imported?

```suggestion
    return failedFuture(new UnprocessableEntityException(
```

Reply:
Sure, updated.